### ISSUE TITLE
why use fc+sigmoid when your classifier is >2 classes?

### ISSUE THREAD

**Issue by rochesterlmg** [lyzustc/Numpy-Implementation-of-ResNet#2]
I guess  fc+softmax is better

**Comment by lyzustc** [comment]
This is true in the case that you are dealing with mutually exclusive classification. But in this project,  our professor told us there must be another class meaning that the input image does not belong to any of given 20 class---so this is exactly a 21-class classification. I have tried use fc+21-class softmax but it seemed not to work better than fc+sigmoid.
